R/spat4cast_submit.R

```diff
@@ -1,8 +1,22 @@
 
-spat4cast_submit <- function(dir, theme = "lai_recovery") {
-  # upload anonymously to submissions bucket
+spat4cast_submit <- function(dir, theme = "lai_recovery", fire = "august_complex") {
+  
+  submitted_files <- list.files(dir, full = TRUE)
+  
+  # Check file type
+  assertthat::assert_that(n_distinct(tools::file_ext(submitted_files)) == 1)
+  assertthat::assert_that(unique(tools::file_ext(submitted_files)) == "tif", msg = "Error: The file extension in the directory is not 'tif'. Please check the file format.")
```

Comment:
looks great. :tada:

Reply:
Oh, we probably want a dictionary of valid site_id names (ideally this will come from some spatial vector table we give people with `site_id` , date of fire/disturbance, and the polygon geometry).